CommandDotNet.Tests/FeatureTests/Arguments/DefaultArgumentMode.cs

```diff
@@ -1,129 +1,221 @@
 using CommandDotNet.Attributes;
 using CommandDotNet.Models;
 using CommandDotNet.Tests.ScenarioFramework;
+using Xunit;
 using Xunit.Abstractions;
 
 namespace CommandDotNet.Tests.FeatureTests.Arguments
 {
-    public class DefaultArgumentMode : ScenarioTestBase<DefaultArgumentMode>
+    public class DefaultArgumentMode : TestBase
     {
-        private static readonly AppSettings OperandMode = TestAppSettings.BasicHelp.Clone(a => a.MethodArgumentMode = ArgumentMode.Parameter);
+        private static readonly AppSettings OperandMode = TestAppSettings.BasicHelp.Clone(a => a.MethodArgumentMode = ArgumentMode.Operand);
         private static readonly AppSettings OptionMode = TestAppSettings.BasicHelp.Clone(a => a.MethodArgumentMode = ArgumentMode.Option);
+        private static readonly AppSettings DeprecatedParameterMode = TestAppSettings.BasicHelp.Clone(a => a.MethodArgumentMode = ArgumentMode.Parameter);
```

Comment:
this test verifies the obsoleted `ArgumentMode.Parameter` and `ArgumentAttribute` still work.

Reply:
The rest of the changes will only impact people with: custom help providers, invokers or argument transformations